From weiyi.huang@gmail.com  Thu Sep 11 02:32:59 2008
From: Huang Weiyi <weiyi.huang@gmail.com>
Date: Sat, 23 Aug 2008 13:56:30 +0800
Subject: USB: remove unused #include <version.h>
To: gregkh@suse.de, greg@kroah.com
Cc: linux-usb@vger.kernel.org
Message-ID: <20080823133422.128C.WEIYI.HUANG@gmail.com>

From: Huang Weiyi <weiyi.huang@gmail.com>

The driver(s) below do not use LINUX_VERSION_CODE nor KERNEL_VERSION.
  drivers/usb/gadget/pxa27x_udc.c

This patch removes the said #include <version.h>.

Signed-off-by: Huang Weiyi <weiyi.huang@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/pxa27x_udc.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/gadget/pxa27x_udc.c
+++ b/drivers/usb/gadget/pxa27x_udc.c
@@ -22,7 +22,6 @@
 #include <linux/module.h>
 #include <linux/kernel.h>
 #include <linux/types.h>
-#include <linux/version.h>
 #include <linux/errno.h>
 #include <linux/platform_device.h>
 #include <linux/delay.h>
